flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/catalog/ExternalCatalogTableBuilder.scala

```diff
@@ -52,7 +52,10 @@ import org.apache.flink.table.descriptors._
   * }}}
   *
   * @param connectorDescriptor Connector descriptor describing the external system
+  *
+  * @deprecated use [[CatalogTableBuilder]]
   */
+@Deprecated
```

Comment:
Do we need two annos for deprecation?

Reply:
`@Deprecated` is for java, `@deprecated` is for scala